cmd/explaintest/r/explain_easy.result

```diff
@@ -87,12 +87,12 @@ TableReader_7	0.33	root	data:Selection_6
 explain select sum(t1.c1 in (select c1 from t2)) from t1;
 id	count	task	operator info
 StreamAgg_12	1.00	root	funcs:sum(col_0)
-└─Projection_35	10000.00	root	cast(5_aux_0)
-  └─MergeJoin_28	10000.00	root	left outer semi join, left key:test.t1.c1, right key:test.t2.c1
-    ├─TableReader_19	10000.00	root	data:TableScan_18
-    │ └─TableScan_18	10000.00	cop	table:t1, range:[-inf,+inf], keep order:true, stats:pseudo
-    └─IndexReader_23	10000.00	root	index:IndexScan_22
-      └─IndexScan_22	10000.00	cop	table:t2, index:c1, range:[NULL,+inf], keep order:true, stats:pseudo
+└─Projection_21	10000.00	root	cast(5_aux_0)
+  └─HashLeftJoin_18	10000.00	root	left outer semi join, inner:TableReader_17, other cond:eq(test.t1.c1, test.t2.c1)
```

Comment:
In this case, its performance may be worse than keep it as apply?

Reply:
If it is an apply, we still cannot use `t1.c1` as key to retrieve rows in `t2` using index on `t2.c1`, because that would filter out rows with `t2.c1 is null`, so the performance of apply would not be better?